roles/foreman_proxy_content/tasks/upgrade.yml

```diff
@@ -1,16 +1,16 @@
 ---
 - include_tasks: certs_generate.yml
 
+- set_fact:
+    foreman_installer_options_internal_use_only:
```

Comment:
This just wipes the internal options. The upgrade pipeline also sets these so I'm not sure that's wise.